deploy-teuthology

```diff
@@ -18,7 +18,7 @@ WORKERS=10
 TARGETS=20
 TEUTH_CONF=${TEUTH_CONF:-"conf/teuthology.cfg"}
 TEUTH_VARS=${TEUTH_VARS:-"teuthology-vars.yml"}
-TEUTH_IMAGE=${TEUTH_IMAGE:-"openSUSE-Leap-15.1-JeOS.x86_64-15.1.0-OpenStack-Cloud-Snapshot9.166"}
+TEUTH_IMAGE=${TEUTH_IMAGE:-"openSUSE-Leap-15.1-JeOS.x86_64-15.1.0-OpenStack-Cloud-Snapshot8.11.7"}
```

Comment:
No, it is very hard to track image version with names in openstack, also there is no symlinks like in regular filesystem. It can be changed later. For now, I've uploaded the same image to OVH and ECP, and tested both. You can always override TEUTH_IMAGE to use any image you want.

Reply:
So there is no way to have it not breaking after new version comes out?

Reply:
Why are we not using the same image that we use for our internal teuhology runs?

Reply:
@toabctl this change exactly to make the image the same ;-)

Reply:
ah. This is the image to run teuthology on - not to run jobs on it. sorry for the noise.